.circleci/config.yml

```diff
@@ -1,8 +1,53 @@
 version: 2.1
 
+orbs:
+  go: circleci/go@1.7.1
+
+parameters:
+  kustomize-version:
+    type: string
+    default: v4.5.7
+
 jobs:
-  build:
+  build-and-push:
     docker:
-     - image: cimg/base:2022.05
+     - image: cimg/base:2022.08
+    resource_class: medium
     steps:
-        - run: echo "Sample CircleCI job."
+      - go/install:
+          version: "1.17"
+      - run:
+          name: "Install kustomize"
```

Comment:
What is this being used for? I didn't find references on this script.

Reply:
this is required by the seldon make scripts that are being called by `./domino-build.sh`